sdk/extensions/Microsoft.Azure.WebJobs.Extensions.Clients/tests/shared/TestHelpers.cs

```diff
@@ -56,6 +58,24 @@ public static TOptions GetConfiguredOptions<TOptions>(Action<IWebJobsBuilder> co
             return options;
         }
 
+        /// <summary>
+        /// Converts an options instance into a dictionary that can be passed into AddInMemoryCollection by
+        /// calling the options Format method, parsing into a JObject, and then collecting the child tokens of the
+        /// JObject.
+        /// </summary>
+        /// <param name="options">The options instance to convert.</param>
+        /// <param name="extensionPath">The extension path for the extension.</param>
+        /// <returns>A dictionary containing the configuration.</returns>
+        public static Dictionary<string, string> ToInMemoryCollection(this IOptionsFormatter options, string extensionPath)
```

Comment:
Do you think we can use the ConfigurationBuilder and https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.configuration.jsonconfigurationextensions.addjsonstream?view=dotnet-plat-ext-5.0?